mod_geoip.c

```diff
@@ -483,44 +489,6 @@ static int geoip_header_parser(request_rec * r)
         }
     }
 
-/* this block should be removed! */
-#if 1
```

Comment:
Memory is made from "child_init" If you are not modified in the request.

because we do not need to be initialized in child_init or server_init.